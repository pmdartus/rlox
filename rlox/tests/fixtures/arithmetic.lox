print "Operators";
print 4 + 2;
print 4 - 2;
print 4 * 2;
print 4 / 2;

print "Precedence";
print 4 * 2 + 1;
print 1 + 4 * 2;

print "Grouping";
print (1 + 4) * 2;
print 2 * (1 + 4);

print "Float";
print 1.5 + 2.5;
print 1.5 - 0.5;
print 0.33 * 3;
print 1 / 3;