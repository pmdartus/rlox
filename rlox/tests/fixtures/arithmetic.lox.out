Operators
6
2
8
2
Precedence
9
9
Grouping
10
10
Float
4
1
0.99
0.33333334
